extra/message_hexdump.go

```diff
@@ -0,0 +1,110 @@
+package extra
+
+import (
+
+	"fmt"
+	"reflect"
+	"github.com/skycoin/skycoin/src/daemon/gnet"
+	"github.com/skycoin/skycoin/src/cipher/encoder"
+	"strconv"
+)
+
+func serializeMessage(msg gnet.Message) []byte {
+	t := reflect.ValueOf(msg).Elem().Type()
+	msgID := gnet.MessageIDMap[t]
+	//if !succ {
```

Comment:
Remove unnecessary comments .

Reply:
Besides , [travis-ci build seems to be failing](https://travis-ci.org/skycoin/skycoin/builds/338800056?utm_source=github_status&utm_medium=notification)